tests/functional/adapter/simple_seed/test_simple_seed.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from dbt.tests.adapter.seed.test_seed import (
+    BasicSeedTests,
+    SeedConfigFullRefreshOn,
+    SeedConfigFullRefreshOff,
+    SeedCustomSchema,
+    SimpleSeedEnabledViaConfig,
+    SeedParsing,
+    SimpleSeedWithBOM,
+    SeedSpecificFormats
+)
+from dbt.tests.adapter.seed.test_seed_type_override import SimpleSeedColumnOverride
+from fixtures_local import properties__schema_yml
```

Comment:
Every db requires different tests. Rather than have the fixture at the top of this file, I preferred to have a local fixtures file for overriding. Just seemed cleaner to me that way.